### PR TITLE
some proofreading

### PR DESCRIPTION
**Pull request by kberov**

### CODE REVIEW

File: lib/Mojolicious/Plugin/AssetPack/Manual/Assets.pod [hunk 1]
```diff
@@ -29,14 +29,14 @@ Here is the basic way of defining an asset:
...
+while the list of files after (foo.js and bar.js) are files that exist in
 one of the public L<static|Mojolicious/static> directories defined in your
 application.
 
 =head1 Online assets
 
-The assets does not need to be locally. They can also be
-L<fetched|Mojolicious::Plugin::AssetPack/fetch> online:
+The assets does not need to be on the same machine. They can also be
+L<fetched|Mojolicious::Plugin::AssetPack/fetch> from anywhere:
```
Comment (jhthorsen) [inline lib/Mojolicious/Plugin/AssetPack/Manual/Assets.pod]: "anywhere" is a bit generic. Not any better than "online" imo. AssetPack can fetch from http and https. How about something like "Assets can be fetched from the world wide web".

File: lib/Mojolicious/Plugin/AssetPack/Manual/Include.pod [hunk 2]
```diff
@@ -31,8 +33,8 @@ L<asset|Mojolicious::Plugin::AssetPack/asset> helper:
 
   %= asset "my-asset.css"
 
-This will result in a single html link tag in production mode, and a list of
-link tags in development mode.
+This will result in a list of link tags in development mode and a single html
+link tag in production mode.
```
Comment (jhthorsen) [inline lib/Mojolicious/Plugin/AssetPack/Manual/Include.pod]: Awesome.

Maybe it should also say ```C<link>``` as to indicate that "link" is code? I think the same goes other places in this document.

File: lib/Mojolicious/Plugin/AssetPack/Manual/Include.pod [hunk 3]
```diff
@@ -24,6 +24,8 @@ other formats as well, such as JavaScript assets.
 
 =head1 USAGE
 
+Here are some ways to use L<Mojolicious::Plugin::AssetPack>.
```
Comment (jhthorsen) [inline lib/Mojolicious/Plugin/AssetPack/Manual/Include.pod]: :+1:

File: lib/Mojolicious/Plugin/AssetPack/Manual/Assets.pod (hunk 1 above)
Comment (kberov) [inline lib/Mojolicious/Plugin/AssetPack/Manual/Assets.pod]: ok, will change it

**Pull request closed without merging**